handler/handler.go

```diff
@@ -118,6 +132,30 @@ func (h *InstanceComplete) Handle(ctx context.Context, workerID int, msg kafka.M
 	return nil
 }
 
+func (h *InstanceComplete) getFilterInfo(ctx context.Context, filterID string, logData log.Data) ([]string, string, error) {
+	dimensions, _, err := h.filters.GetDimensions(ctx, "", h.cfg.ServiceAuthToken, "", filterID, nil)
+	if err != nil {
+		return nil, "", &Error{
+			err:     fmt.Errorf("failed to get dimensions: %w", err),
```

Comment:
we should be moving to use errors.Wrap (from github.com/pkg/errors) rather than fmt.Errorf now, as this preserves the stack trace

Reply:
```suggestion
			err:     errors.Wrap(err, "failed to get dimensions"),
```